.github/workflows/dev-cluster-check.yml

```diff
@@ -40,6 +40,18 @@ jobs:
         with:
           maven-command: clean verify -Pcucumber -Dparallel -Devent-bridge.manager.url=$DEV_CLUSTER_MANAGER_URL -Dkeycloak.realm.url=https://sso.redhat.com/auth/realms/redhat-external -Dbridge.client.id=$DEV_SSO_CLIENT_ID -Dbridge.client.secret=$DEV_SSO_CLIENT_SECRET
           working-directory: integration-tests
+      - name: Send mail with link to failing job
+        uses: dawidd6/action-send-mail@v3
+        if: ${{ failure() }}
+        with:
+          server_address: smtp.gmail.com
+          server_port: 465
+          username: ${{secrets.CI_MAIL_USERNAME}}
```

Comment:
I would suggest to add secrets also to "Add mask" step to be 100% sure that it will not leak anyhow.

Reply:
Done